app/views/layouts/application.html.erb

```diff
@@ -20,18 +23,18 @@
 </head>
 <body>
 
-<script type="text/javascript">
-  var lock = new Auth0Lock('<%= Rails.configuration.auth0_client_id %>', '<%= Rails.configuration.auth0_domain %>');
-  function signin() {
-    lock.show({
-      dict: '<%= I18n.locale %>',
-      icon: '/logo-alt.png',
-      callbackURL: '<%= auth_auth0_callback_url %>', responseType: 'code', authParams: {
-        scope: 'openid profile'
-      }
-    });
-  }
-</script>
+<% current_mode.if_online do %>
+  <script type="text/javascript">
+    var lock = new Auth0Lock('<%= Rails.configuration.auth0_client_id %>', '<%= Rails.configuration.auth0_domain %>');
+    function signin() {
+      lock.show({
```

Comment:
Be aware of keeping the locale an logo. 

it seems it got lost along the merge